README.md

```diff
@@ -650,6 +650,8 @@ Table of Contents
 
 ## Email
 
+  * [Migomail(https://migomail.com/) – migomail provides transactional smtp server.
+  * [Mailcot(https://mailcot.com/) – mailcot provides email marketing services.
```

Comment:
please update with details of the free tier, does not seem to be a free service at all.